compute/cker/src/train/AvgPool.test.cc

```diff
@@ -0,0 +1,286 @@
+     *
+     *  10(0)  15(1)  2(2)
+     *  10(3)  12(4)  17(5)   -(forward)->     16(0)
+     *  50(6)  30(7)  -2(8)
+     *
+     *
+     *  depth[1]
+     *  input(index):                      output(index):
+     *
+     *  -1(0)  2(1)  3(2)
+     *  8(3)   9(4)  2(5)    -(forward)->       4(0)
+     *  4(6)   2(7)  7(8)
+     */
+
+    std::vector<float> input(in.FlatSize());
+    auto input_mat = MapAsMatrixWithLastDimAsRows(input.data(), in);
+    input_mat << /* depth0 */ 10, 15, 2, 10, 12, 17, 50, 30, -2,
+      /* depth1 */ -1, 2, 3, 8, 9, 2, 4, 2, 7;
+    std::vector<float> expected_output = {16, 4};
+    verifier.verifyForward(input, expected_output);
+
+    /**
+     * depth[0]
+     * ouput_deriv:                 input_deriv:
+     *
+     *                             0.02  0.02  0.02
+     *    0.18     -(backward)->   0.02  0.02  0.02
+     *                             0.02  0.02  0.02
+     *
+     *
+     * depth[1]
+     * output_deriv:                input_deriv:
+     *                              0.04  0.04  0.04
+     *    0.36     -(backward)->    0.04  0.04  0.04
+     *                              0.04  0.04  0.04
+     */
+
+    std::vector<float> output_deriv = {0.18, 0.36};
+    std::vector<float> expected_input_deriv(in.FlatSize());
+    auto input_deriv_mat = MapAsMatrixWithLastDimAsRows(expected_input_deriv.data(), in);
+    input_deriv_mat << /* depth0 */ 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02,
+      /* depth1 */ 0.04, 0.04, 0.04, 0.04, 0.04, 0.04, 0.04, 0.04, 0.04;
+    verifier.verifyBackward(output_deriv, expected_input_deriv);
+  }
+
+  // with padding case
+  {
+    nnfw::cker::PoolParams op_param;
+    {
+      op_param.stride_height = 2;
+      op_param.stride_width = 2;
+      op_param.filter_height = 2;
+      op_param.filter_width = 2;
+      op_param.padding_values.height = 2;
+      op_param.padding_values.width = 2;
+    }
+    nnfw::cker::Shape in = {1, 2, 2, 1};
+    nnfw::cker::Shape out = {1, 3, 3, 1};
+
+    AvgPoolOpVerifier<float> verifier(op_param, in, out);
+
+    /**
+     * input_with_padding:             expected_output:
+     *
+     *    4   8                              0   0   0
+     *    9   2            -(forward)->      0  5.75 0
+     *                                       0   0   0
+     */
+
+    std::vector<float> input = {4, 8, 9, 2};
+    std::vector<float> expected_output = {0, 0, 0, 0, 5.75, 0, 0, 0, 0};
+    verifier.verifyForward(input, expected_output);
+
+    /**
+     * output_deriv:                    input_deriv:
+     *
+     *  0.1   0.1   0.1                     0.1   0.1
+     *  0.1   0.4   0.3   -(backward)->     0.1   0.1
+     *  0.5   0.1   0.1
+     */
+    std::vector<float> output_deriv = {0.1, 0.1, 0.1, 0.1, 0.4, 0.3, 0.5, 0.1, 0.1};
+    std::vector<float> expected_input_deriv = {0.1, 0.1, 0.1, 0.1};
+    verifier.verifyBackward(output_deriv, expected_input_deriv);
+  }
+}
+
+TEST(CKer_Operation, neg_AvgPool)
```

Comment:
Could you change the test name to a specific name like the below?
```suggestion
TEST(CKer_Operation, neg_AvgPoolInvalidExpectedValue)
```

Reply:
> Could you change the test name to a specific name like the below?

Ok, I'll change it.